src/utils/errors.js

```diff
@@ -14,30 +14,26 @@
  * limitations under the License.
  */
 
-const {
-  createAbortError,
-  isAbortError,
-} = require('web-activities/activity-ports');
-
 /**
  * Whether the specified error is an AbortError type.
  * See https://heycam.github.io/webidl/#aborterror.
  * @param {*} error
  * @return {boolean}
  */
 export function isCancelError(error) {
```

Comment:
Could this function be called on an error generated by web-activities (in which case the logic would not be correct)? I'm not sure this simplification is worth it because it could lead to unexpected behavior, since there are now diverging implementations of this functionality here and in web-activities.

Reply:
Good news, it works with `web-activities` abort errors. I just added a test verifying this.
https://github.com/subscriptions-project/swg-js/pull/2662/files#diff-08d49f0abfc63df34bf11282639d901e155eda0210bf852990daeffa102be431R28

Reply:
Oh, OK, cool.